unified_planning/io/ma_pddl_writer.py

```diff
@@ -287,58 +289,11 @@ def _write_domain(self):
                         "MA-PDDL supports only boolean and numerical fluents"
                     )
 
-            predicates_agent_goal = []
-            functions_agent_goal = []
-            for g in self.problem.goals:
-                if g.is_dot():
-                    f = g.args[0].fluent()
-                    agent = g.agent()
-                    # args = g.args
-                    # objects = g.args[0].args
-                    if f not in ag.fluents:
-                        if f.type.is_bool_type():
-                            params = []
-                            i = 0
-                            for param in f.signature:
-                                if param.type.is_user_type():
-                                    params.append(
-                                        f" {self._get_mangled_name(param)} - {self._get_mangled_name(param.type)}"
-                                    )
-                                    i += 1
-                                else:
-                                    raise UPTypeError(
-                                        "MA-PDDL supports only user type parameters"
-                                    )
-                            predicates_agent_goal.append(
-                                f'(a_{self._get_mangled_name(f)} ?agent - {self._get_mangled_name(agent) + "_type"}{"".join(params)})'
-                            )
-                        elif f.type.is_int_type() or f.type.is_real_type():
-                            params = []
-                            i = 0
-                            for param in f.signature:
-                                if param.type.is_user_type():
-                                    params.append(
-                                        f" {self._get_mangled_name(param)} - {self._get_mangled_name(param.type)}"
-                                    )
-                                    i += 1
-                                else:
-                                    raise UPTypeError(
-                                        "MA-PDDL supports only user type parameters"
-                                    )
-                            functions_agent_goal.append(
-                                f'(a_{self._get_mangled_name(f)} ?agent - {self._get_mangled_name(agent) + "_type"}{"".join(params)})'
-                            )
-                        else:
-                            raise UPTypeError(
-                                "MA-PDDL supports only boolean and numerical fluents"
-                            )
-
```

Comment:
@Alee08 Why did you remove this code? It seems that this change broke the integration with FMAP.